documentation/modules/exploit/multi/http/lesplunk_upload_app_exec.md

```diff
@@ -0,0 +1,83 @@
+## Vulnerable Application
+
+This module exploits a feature of Splunk whereby a custom application can be
+uploaded through the web based interface. Through the \'script\' search command a
+user can call commands defined in their custom application which includes arbitrary
+perl or python code. To abuse this behavior, a valid Splunk user with the admin
+role is required. By default, this module uses the credential of "admin:changeme",
+the default Administrator credential for Splunk. Note that the Splunk web interface
+runs as SYSTEM on Windows, or as root on Linux by default. This module has been
+tested successfully against Splunk 5.0, 6.1, 6.1.1 and 7.2.4.
+Version 7.2.4 has been tested successfully against OSX as well.
+
+## Verification Steps
+
+  1. Start msfconsole
+  2. Do: ```use exploit/multi/http/splunk_upload_app_exec```
+  3. Set required variables (you will need admin credentials)
+  4. Do: ```SET TARGET 1```
+  5. Do: ```SET RHOST 192.168.1.1```
+  6. Do: ```SET USERNAME admin```
+  7. Do: ```SET PASSWORD changeme```
+  8. You should get a reverse shell on port 4444 via the predefined handler.
+
+## External Demo
+[First PoC](http://blog.7elements.co.uk/2012/11/splunk-with-great-power-comes-great-responsibility.html) <br>
```

Comment:
`<br>` should not be necessary for markdown. Consider using a list with `*` preceding each link.

Reply:
thanks, done!